.circleci/config.yml

```diff
@@ -52,11 +52,9 @@ jobs:
           command: |
             echo "//registry.npmjs.org/:_authToken=${NPM_KEY}" > ~/.npmrc
             echo 'Publishing to npmjs.org'
-            npm deprecate -f '@cennznet/api@1.3.0-alpha.1'
-            npm deprecate -f '@cennznet/util@1.3.0-alpha.1'
-            npm deprecate -f '@cennznet/types@1.3.0-alpha.1'
-            # commenting the following command for now to unpublish the earlier version
-            # npx lerna exec -- 'cd build && npm publish --access public'
+            yarn
```

Comment:
It would be better to run build in the `package` step before copying in the files
```
            yarn && yarn build
            npm install cpx
            npx lerna exec -- 'cpx package.json build/'
            npx lerna exec -- 'cpx README.md build/'
            npx lerna exec -- 'cpx ../../LICENSE build/'
```